docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -464,3 +464,7 @@ The example below would result in a live stream where assets `source_1.m3u8`, `s
   }
 }
 ```
+
+## CONTINUOUS CAPTURE
+
+The Task Engine [VOD Capture](TaskEngineWorkflows.md#vod-capture) workflow supports the `continuous_capture` parameter which will begin capturing clips before the specified end time. By default it will begin capturing once 80% of the time has elapsed between the earliest start time of the specified clips and the latest end time. This can be overriden by setting the `run_at` parameter.
```

Comment:
```suggestion
The Task Engine [VOD Capture](TaskEngineWorkflows.md#vod-capture) workflow supports the `continuous_capture` parameter which will begin capturing clips before the specified end time. By default, it will begin capturing once 80% of the content is available, i.e. at 80% of the way from the earliest start time to the latest end time of all clips. This can be overridden by setting the `run_at` parameter.
```

Small phrasing adjustment, I think it helps clarity.